khipu-metabolomics/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "khipu-metabolomics" %}
+{% set version = "0.7.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/khipu-metabolomics-{{ version }}.tar.gz
+  sha256: 5c89c28d30fccefb1e468d8fbcb3637bd452f002438291f7eb910a762ceb99f1
+
+build:
+  entry_points:
+    - khipu=khipu.command_line:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  run_exports:
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - mass2chem
+    - treelib
+    - networkx
+    - pandas
+test:
+  imports:
+    - khipu
+  commands:
+    - pip check
+    - khipu --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/shuzhao-li/khipu
+  summary: Common utilities for interpreting mass spectrometry data
+  license: BSD-1-Clause
```

Comment:
It's BSD 3 in the file https://github.com/shuzhao-li-lab/khipu/blob/main/LICENSE

Reply:
Noted - already updated.